docs/INTEGRATION.md

```diff
@@ -79,7 +79,27 @@ Add the following job to your project's `.drone.yml` pipeline:
       - hadolint --version
       - hadolint Dockerfile
 ```
-
+## Jenkins declarative pipeline
+
+You can add a step during your CI proccess to lint and archive the output of hadolint
+
+```groovy
+stage ("lint dockerfile") {
+    agent {
+        docker {
+            image 'hadolint/hadolint:latest-debian'
+        }
+    }
+    steps {
+        sh 'hadolint dockerfiles/* | tee -a hadolint_lint.txt'
+    }
+    post {
+        always {
+            archiveArtifacts 'hadolint_lint.txt'
```

Comment:
along with the suggestion above publish the check style report with warnings-ng plugin
```suggestion
            archiveArtifacts 'hadolint_lint.xml'
            recordIssues blameDisabled: true, enabledForFailure: true, tools: [checkStyle(pattern: 'hadolint_lint.xml')]

```